### PR TITLE
chore: JSDoc on warp-element

### PR DESCRIPTION
**Pull request by leftieFriele**
First stab at JSDoc on this, would this be sufficient? I'm a newbie when it comes to applying type information using JSDoc, so 🙏 ❤️ 

I also removed export from some of the functions in `utils.js`, as they didn't appear to make sense outside the module and that having them private as "default" might be better.

### CODE REVIEW

File: warp-element/src/utils.js [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
     if (str.toLocaleLowerCase().includes('blocket')) return { sld: 'blocket', tld: 'se' };
     if (str.toLocaleLowerCase().includes('tori')) return { sld: 'tori', tld: 'fi' };
     if (str.toLocaleLowerCase().includes('finn')) return { sld: 'finn', tld: 'no' };
     if (str.toLocaleLowerCase().includes('dba')) return { sld: 'dba', tld: 'dk' };
     return { sld: 'finn', tld: 'no' };
 };
 
+
+/*
+ * @param {string} [brandStr] - a string with brand information
```
Comment (leftieFriele) [inline warp-element/src/utils.js:17]: Should this be "an URL" or something like that?

Comment (digitalsadhu) [reply warp-element/src/utils.js:17]: I think string is correct

File: warp-element/src/utils.js [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
+ * @returns {import("./global.js").Brand} brand object
```
Comment (digitalsadhu) [inline warp-element/src/utils.js:18]: So this works? You can reference the .Brand property because there is a @typedef for Brand in global.js then?

Comment (leftieFriele) [reply warp-element/src/utils.js:18]: As far as I've understood [import types](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#import-types) that is how it works.

Comment (digitalsadhu) [reply warp-element/src/utils.js:18]: yea, might well be by the looks of it though slightly hard to tell from those docs but if its working its working so yay.

**Review by digitalsadhu — APPROVED** [review]
Noice

**Pull request merged**